gtest/src/error.rs

```diff
@@ -88,3 +88,13 @@ pub enum TestError {
     #[display(fmt = "Reading of program state failed: `{_0}`")]
     GbuildArtifactNotFound(String),
 }
+
+macro_rules! _user_panic {
```

Comment:
Moreover, it's more like "misuse" of gtest api: so we can even say that we panicked because of users wrong usage of library

Reply:
So what naming do you suggest here?

Reply:
The idea here is that we use user-facing panics to communicate gtest API usage errors to the user.

Maybe `api_misuse_panic` would be a better name?

Reply:
Or `usage_panic`.

Reply:
Discussed, fixed